rllib/agents/sac/sac_policy.py

```
@@ -41,12 +40,14 @@ def build_sac_model(policy, obs_space, action_space, config):
             "using a Tuple action space, or the multi-agent API.")
 
     if config["use_state_preprocessor"]:
-        default_model = None  # catalog decides
         num_outputs = 256  # arbitrary
         config["model"]["no_final_linear"] = True
     else:
-        default_model = NoopModel
         num_outputs = int(np.product(obs_space.shape))
+    # Force-ignore any additionally provided hidden layer sizes.
+    # Everything should be configured using SAC's "Q_model" and "policy_model"
+    # settings.
+    config["model"]["fcnet_hiddens"] = []
```

Should we raise an error?